src/Logging.AzureAppServices/src/WebAppContext.cs

```diff
@@ -18,7 +18,7 @@ internal sealed class WebAppContext : IWebAppContext
     private WebAppContext() { }
 
     /// <inheritdoc />
-    public string HomeFolder { get; } = Environment.GetEnvironmentVariable("HOME");
+    public string HomeFolder { get; } = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
```

Comment:
This is not correct. The "HOME" environment variable has a special meaning in App Service where this is used: https://docs.microsoft.com/en-us/azure/app-service/reference-app-settings?tabs=kudu%2Cdotnet